app/build.gradle

```diff
@@ -22,14 +23,45 @@ android {
       proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
     }
   }
+  compileOptions {
+    sourceCompatibility = "1.8"
+    targetCompatibility = "1.8"
+  }
+  kotlinOptions {
+    jvmTarget = '1.8'
+  }
+  packagingOptions {
+    exclude 'META-INF/*'
+  }
 }
 
 dependencies {
   implementation fileTree(dir: 'libs', include: ['*.jar'])
+
+  // Kotlin
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+  implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.4'
+
+  // Android
   implementation 'androidx.appcompat:appcompat:1.1.0'
   implementation 'androidx.core:core-ktx:1.2.0'
+  implementation 'com.google.android.material:material:1.2.0-alpha05'
+  implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.2.0'
+  implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.2.0'
+
+  // Dependency Injection
+  api 'com.google.dagger:dagger:2.26'
+  kapt 'com.google.dagger:dagger-compiler:2.26'
+
+  // Logging
+  implementation 'com.jakewharton.timber:timber:4.7.1'
+
+  // Testing
   testImplementation 'junit:junit:4.13'
```

Comment:
Should we drop JUnit4 now that we're integrating JUnit5 in this PR?

Reply:
For instrumented tests JUnit5 is experimental and only with some added libraries. But I think JUnit5 is good to go for unit tests.